Jetcaster/wear/src/main/java/com/example/jetcaster/ui/library/LatestEpisodesScreen.kt

```diff
@@ -138,36 +179,17 @@ fun MediaContent(
 @OptIn(ExperimentalHorologistApi::class)
 @Composable
 fun ButtonsContent(
-    viewState: LatestEpisodeViewState,
-    onChangeSpeedButtonClick: () -> Unit,
+    episodes: List<EpisodeToPodcast>,
     onPlayButtonClick: () -> Unit,
-    onPlayEpisode: (PlayerEpisode) -> Unit
+    onPlayEpisode: (PlayerEpisode) -> Unit,
 ) {
-
-    Row(
-        modifier = Modifier
-            .padding(bottom = 16.dp)
-            .height(52.dp),
-        verticalAlignment = Alignment.CenterVertically,
-        horizontalArrangement = Arrangement.spacedBy(6.dp, Alignment.CenterHorizontally),
-    ) {
-        Button(
-            imageVector = ImageVector.vectorResource(R.drawable.speed),
-            contentDescription = stringResource(id = R.string.speed_button_content_description),
-            onClick = { onChangeSpeedButtonClick() },
-            modifier = Modifier
-                .weight(weight = 0.3F, fill = false),
-        )
-
-        Button(
-            imageVector = Icons.Filled.PlayArrow,
-            contentDescription = stringResource(id = R.string.button_play_content_description),
-            onClick = {
-                onPlayButtonClick()
-                onPlayEpisode(viewState.libraryEpisodes[0].toPlayerEpisode())
-            },
-            modifier = Modifier
-                .weight(weight = 0.3F, fill = false),
-        )
-    }
+    Chip(
+        label = stringResource(id = R.string.button_play_content_description),
+        onClick = {
+            onPlayButtonClick()
+            onPlayEpisode(episodes[0].toPlayerEpisode())
```

Comment:
Should this play all items?

Reply:
yes, this play is on top of the latest episodes list and expectation is that all of them will be played (added to queue)

Reply:
Why just episodes[0] then?

Reply:
you are right, that was a bug. Fixed.